src/assets/styles/components/_cards.scss

```diff
@@ -55,34 +56,167 @@
 
 		p {
 			font-weight: 400;
+			line-height: 130%;
+		}
+
+		.card__content {
+			block-size: 100%;
+			inline-size: 100%;
+			left: 0;
+			position: absolute;
+			top: 0;
```

Comment:
You can write `inset: 0;` instead of using `left` and `top`, see: https://developer.mozilla.org/en-US/docs/Web/CSS/inset

Reply:
Nice, that's cleaner 👍